### DISCUSSION TITLE
error connecting your engraver

### DISCUSSION THREAD

**Discussion by ghost** [arkypita/LaserGRBL#2057] [General]
I keep getting this what did I miss?
![Screenshot 2023-04-24 115513](https://user-images.githubusercontent.com/131412897/234065936-7dc9b460-f742-4bd6-8473-e6aa38d61516.png)

**Comment by StuartB4** [comment]
Are you sure you have the correct baud rate for your machine. 
I can get that message if I pick the incorrect number. 

![Untitled](https://user-images.githubusercontent.com/16980026/234112285-9dd6da9a-38ed-470e-94b2-0d84225dfef9.png)

**Reply by krist3815** [reply]
I’m getting the same error. I’ve tried changing the baud rate but I’m not having any luck.

**Comment by Tribo-2dot0** [comment]
Qual sua máquina?
